src/libs/actions/Session/index.js

```diff
@@ -348,7 +348,7 @@ function setPassword(password, validateCode, accountID) {
             }
 
             // This request can fail if the password is not complex enough
-            Onyx.merge(ONYXKEYS.ACCOUNT, {error: response.message});
+            Onyx.merge(ONYXKEYS.ACCOUNT, {errors: {[DateUtils.getMicroseconds()]: response.message}});
```

Comment:
Can't we use Localize.translateLocal here?

Reply:
We could. One way I can think of is to map error codes with localize phrases and get the appropriate phrase when we receive the response code from the mapping.
But since, we're deprecating this API call soon, I think we can skip this for now.